AI-and-Analytics/Getting-Started-Samples/README.md

```diff
@@ -0,0 +1,34 @@
+# Getting Started Samples for Intel® oneAPI AI Analytics Toolkit
+
+The Intel® oneAPI AI Analytics Toolkit gives data scientists, AI developers, and researchers familiar Python* tools and frameworks to accelerate end-to-end data science and analytics pipelines on Intel® architectures. The components are built using oneAPI libraries for low-level compute optimizations. This toolkit maximizes performance from preprocessing through machine learning, and provides interoperability for efficient model development.
+
+You can find more information at [ AI Analytics Toolkit](https://software.intel.com/content/www/us/en/develop/tools/oneapi/ai-analytics-toolkit.html).
+
+Users could understand how to run samples for different components in oneAPI AI Analytics Toolkits with those getting started samples.
+
+## License
+Code samples are licensed under the MIT license. See
+[License.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt) for details.
+
+Third party program Licenses can be found here: [third-party-programs.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/third-party-programs.txt)
+
+# Getting Started Samples
+
+| Compoment      | Folder                                             | Description
```

Comment:
Please organize this by the first column in Alpha order

Reply:
done. thanks.